devops/scripts/upgrade-lock-files.sh

```diff
@@ -7,10 +7,8 @@ list="$(find . -type f -name "*.hcl")"
 
 for i in $list
 do
-    rm $i
-    echo "Updating lock file" + $i
-    dir=$(dirname $i)
+    rm "$i"
+    echo "Updating lock file" + "$i"
```

Comment:
I think this line is wrong anyway.
Should update it to....

`echo "Updating lock file $i"`

Reply:
(it currently echos the `+` character, rather than performing a string concat)